runtime/integration-tests/src/cases/lp/investments.rs

```diff
@@ -703,10 +742,28 @@ mod with_foreign_currency {
 				remaining_amount
 			);
 
-			lp_utils::process_outbound::<T>(|msg| {
+			// Required given that the deposit request address for this account
+			// does not have the extra chain id + EVM tag.
+			let tranche_investor_remote_account =
+				lp::utils::remote_account_of::<T>(Keyring::TrancheInvestor(1));
+			let tranche_investor_evm_account = &tranche_investor_remote_account.as_slice()[..20];
+
+			lp_utils::process_gateway_message::<T>(|msg| {
```

Comment:
This was added on top of what we already had. Needed to check the initial inbound message that is sent.